plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -106,6 +107,10 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 				return r.authorizeGet(nodeName, vaVertexType, attrs)
 			}
 			return authorizer.DecisionNoOpinion, fmt.Sprintf("disabled by feature gate %s", features.CSIPersistentVolume), nil
+		case svcAcctResource:
+			if utilfeature.DefaultFeatureGate.Enabled(features.TokenRequest) {
+				return r.authorizeCreateToken(nodeName, serviceAccountVertexType, attrs)
+			}
```

Comment:
if the feature gate is not enabled, return no decision here (just like volume attachment)

Reply:
Done.